master/getting-started/bare-metal/installation/binary.md

```diff
@@ -36,10 +34,6 @@ Felix has the following pre-requisites:
     check the version, run `ipset --version`.
 -   The conntrack, iptables and ipsets kernel modules must be available
     (or compiled-in).
```

Comment:
From what I understand, if they use `calico/node`, then they don't have to worry about these requirements, it's all there in the container. So that is why we only have it here.

Reply:
Ah good point - partly.  I think that is true for the userspace programs, i.e. iptables, ipset and conntrack-tools.  But I think the kernel-level and module reqts would apply even to calico/node.